varnish2_1_5.rb

```diff
@@ -2,14 +2,9 @@
 
 class Varnish2_1_5 < Formula
   url 'http://repo.varnish-cache.org/source/varnish-2.1.5.tar.gz'
-  head 'http://repo.varnish-cache.org/source/varnish-3.0.0-beta1.tar.gz'
   homepage 'http://www.varnish-cache.org/'
 
-  if ARGV.build_head?
```

Comment:
MD5 is deprecated, removing calls to the function. Also, since HEAD isn't an option, no need for the if statement.